examples/entities.yaml

```diff
@@ -12,6 +12,10 @@ apiVersion: backstage.io/v1alpha1
 kind: Component
 metadata:
   name: example-website
+  annotations:
```

Comment:
I don't think we want to commit this to `main`, just needed for local.

Reply:
Agreed.   Committed in error.